### PR TITLE
Feature/manage home page

### PR DESCRIPTION
**Pull request by anhmht**
Implement ManageHomePage.jsx

 - Create a collection home_page
> Please reset your local db to add this collection
 - Show setting on a list
 - At the end of each row add 'Cập nhật' button to open a pop up
 - Pop up contains a edditor to modify home page content
image

According to this https://ant.design/docs/react/recommendation
so I decide to use react-quill editor

> yarn add react-quill

image

### CODE REVIEW

File: imports/api/methods/homePage.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { check } from 'meteor/check';
+import HomePageWorker from '../../../lib/classes/home/homePage';
+
+Meteor.methods({
+  'homePage.updateSettings': (updatedData) => {
```
Comment (hoadinh2010) [inline imports/api/methods/homePage.js]: Change the variable name  `updatedData`. We need it easier to read (homepageData, homepageObj, ... )

File: imports/ui/modules/home/components/EditHomePageForm.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  }
+
+  handleChange = (value) => {
+    this.setState({ text: value });
+  }
+
+  handleUpdate = () => {
+    const { options } = this.props;
+    const { text } = this.state;
+    const data = {
```
Comment (hoadinh2010) [inline imports/ui/modules/home/components/EditHomePageForm.jsx]: Change the var name. Keep in mind, This code is read by everybody in project and should make it easy to read.

File: imports/ui/modules/home/components/ManageHomePage.jsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  connect(null, mapDispatchToProps),
+  withTracker(() => {
+    const handleSub = Meteor.subscribe('homePage.getSetting');
+    let homePage = null;
+    if (handleSub.ready()) {
+      homePage = HomePages.findOne({});
+    }
+    return {
+      homePage,
+      handleSub
```
Comment (hoadinh2010) [inline imports/ui/modules/home/components/ManageHomePage.jsx]: You return `handleSub`, but don't use it on the component, You must have `isLoading` to show loading for settings

File: imports/api/methods/homePage.js (hunk 1 above)
Comment (anhmht) [reply imports/api/methods/homePage.js]: variable name was changed to homepageData

File: imports/ui/modules/home/components/EditHomePageForm.jsx (hunk 2 above)
Comment (anhmht) [reply imports/ui/modules/home/components/EditHomePageForm.jsx]: variable name was changed to homepageData

File: imports/ui/modules/home/components/ManageHomePage.jsx (hunk 3 above)
Comment (anhmht) [reply imports/ui/modules/home/components/ManageHomePage.jsx]: return isLoading instead

**Pull request merged**